help.html

```diff
@@ -3,14 +3,20 @@
 <h1>Welcome to Searchfox</h1>
 
 <h3>
-<a href="/mozilla-central/source" style="text-decoration: underline;">Direct link to mozilla-central</a>
-| <a href="/mozilla-mobile/source" style="text-decoration: underline;">mozilla-mobile</a>
-| <a href="/comm-central/source" style="text-decoration: underline;">comm-central</a>
-| <a href="/nss/source" style="text-decoration: underline;">nss</a>
-| <a href="/whatwg-html/source" style="text-decoration: underline;">WHATWG HTML spec</a>
-| <a href="/mozilla-beta/source" style="text-decoration: underline;">mozilla-beta</a>
-| <a href="/mozilla-release/source" style="text-decoration: underline;">mozilla-release</a>
-| <a href="/mozilla-esr60/source" style="text-decoration: underline;">mozilla-esr60</a>
+Direct links to:
+<table style="width:100%"><tr><td style="text-align: center; vertical-align: top">
+    <a href="/mozilla-central/source" style="text-decoration: underline;">mozilla-central</a><br/>
+    <a href="/mozilla-mobile/source" style="text-decoration: underline;">mozilla-mobile</a><br/>
+    <a href="/comm-central/source" style="text-decoration: underline;">comm-central</a><br/>
+</td><td style="text-align: center; vertical-align: top">
+    <a href="/nss/source" style="text-decoration: underline;">nss</a><br/>
+    <a href="/whatwg-html/source" style="text-decoration: underline;">WHATWG HTML spec</a><br/>
+</td><td style="text-align: center; vertical-align: top">
+    <a href="/mozilla-beta/source" style="text-decoration: underline;">mozilla-beta</a><br/>
+    <a href="/mozilla-release/source" style="text-decoration: underline;">mozilla-release</a><br/>
+    <a href="/mozilla-esr60/source" style="text-decoration: underline;">mozilla-esr60</a><br/>
+    <a href="/mozilla-esr60/source" style="text-decoration: underline;">mozilla-esr68</a><br/>
```

Comment:
the href still has 60 instead of 68 (the textContent is right though)

Reply:
Hah, nice catch! Updated